### ISSUE TITLE
Allow lists to not be indented

### ISSUE THREAD

**Issue by bkane-msft** [mikefarah/yq#1841]
**Please describe your feature request.**

I wish I could force my YAML output to have a list indentation style of :

```yaml
key:
  value
list:
- one
- two
- three
```

instead of:

```yaml
key:
  value
list:
  - one
  - two
  - three
```

While I'm new to yq, I don't think this is currently possible after reading https://mikefarah.gitbook.io/yq/usage/output-format#indent , skimming the man page, and studying `--help`

> Note that lists are indented at the same level as the map key at indent level 2, but are more deeply indented at indent level 4 and greater. This is (currently) a quirk of the underlying yaml parser

However, I'd be super happy to be proven wrong!

**Describe the solution you'd like**

Not sure, maybe another CLI flag  that pertains to lists or a `style` operator change?

**Describe alternatives you've considered**

Maybe I could write some sort of shell script to post-process my YAML file?

**Additional context**

I want to use yq to format my large [octodns](https://github.com/octodns/octodns) YAML files by top-level key, but keep the pre-existing comments and list indentation. Keeping the comments already seems to work, so that just leaves the list indentation.

I totally understand if this CLI is limited by the underlying YAML lib and so couldn't do this without an unreasonable amount of work, but I didn't see any issues with a [search](https://github.com/mikefarah/yq/issues?q=dis%3Aissue+list+indented), so thought I'd ask.

Thanks so much for your work on `yq` and releasing it to the public!

**Comment by sinicide** [comment]
I think this was noted back in Issue #462 the indentation for list is a result of the v3 yaml libraries. I too would love if it can preserve the indentation format as this is almost a perfect tool for me to use to modify k8s yaml files, but this indentation is killing it for me.

**Comment by bkane-msft** [comment]
Thanks, @sinicide - I'll close this issue and subscribe to that one.

**Comment by mikefarah** [comment]
Fixed in v4.52.1
